worldengine/generation.py

```diff
@@ -105,12 +105,20 @@ def initialize_ocean_and_thresholds(world, ocean_level=1.0):
     """
     e = world.elevation['data']
     ocean = fill_ocean(e, ocean_level)
-    hl = find_threshold_f(e, 0.10)  # the highest 10% of all (!) land are declared hills
-    ml = find_threshold_f(e, 0.03)  # the highest 3% are declared mountains
+
+    # values chosen to emulate the exponential behavior seen here:
+    # http://www.ngdc.noaa.gov/mgg/global/etopo1_surface_histogram.html
+    pl = find_threshold_f(e, 0.95, ocean=ocean)  # the highest x% of land are declared plains; save some for beaches (?)
+    hl = find_threshold_f(e, 0.50, ocean=ocean)  # the highest x% are declared hills
```

Comment:
Looking at the two images above, the scale of the platec-world to Earth seems to be roughly 1000:1. So hills right now are at roughly 1000m - the value would be lower if the first part of the graph wasn't as steep as it is.
You said 50% seemed like a lot. Moving hills *down* will increase that number even further - to maybe 75%.

The graphs can probably be used to find reasonable values. Although the definitions of hills, mountains etc. are kind of arbitrary at the moment.